services/ingestion/etl/inforegio/json/src/lib/transform.js

```diff
@@ -19,7 +20,7 @@ const formatDate = date => {
   const d = date.split(/\//);
```

Comment:
Why do you keep those lines (with `d`)? In the end, you get the date from `date`

Reply:
I think these checks are still valid?
```
  if (d === null || d.length !== 3) return null;
  if (d[2].length === 2) d[2] = `20${d[2]}`;
```